src/Command/DumpCommand.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Phoenix\Command;
+
+use Phoenix\Command\AbstractCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DumpCommand extends AbstractCommand
+{
+    protected function configure()
+    {
+        $this->setName('dump')
+            ->setDescription('Dump actual database structure to migration file')
+            ->addOption('data', 'd', InputOption::VALUE_NONE, 'Dump structure and also data')
+            ->addOption('ignore-tables', null, InputOption::VALUE_OPTIONAL, 'Comma separaterd list of tables to ignore (Structure and data). Default: phoenix_log')
+            ->addOption('ignore-data-tables', null, InputOption::VALUE_OPTIONAL, 'Comma separaterd list of tables which will be exported without data (Option -d, --data is required to use this option)')
+            ->addOption('indent', 'i', InputOption::VALUE_OPTIONAL, 'Indentation. Available values: 2spaces, 3spaces, 4spaces, 5spaces, tab', '4spaces')
+        ;
+
+        parent::configure();
+    }
+
+    protected function runCommand(InputInterface $input, OutputInterface $output)
+    {
+        $ignoredTables = array_map('trim', explode(',', $input->getOption('ignore-tables') . ',' . $this->config->getLogTableName() ? : $this->config->getLogTableName()));
+        $output->writeln('');
+
+        $indent = $this->getIndent($input);
+
+        $migration = '';
+        $tables = $this->getOrderedTables($ignoredTables);
+        foreach ($tables as $table) {
+            $migration .= $indent . "\$this->table('{$table->getName()}')\n";
+            foreach ($table->getColumns() as $column) {
+                $migration .= "$indent$indent" . "->addColumn('{$column->getName()}', '{$column->getType()}')\n";
```

Comment:
Yes, me too :D I just add them here and testing this command now, I will push changes soon